### DISCUSSION TITLE
DATA COLLECTION

### DISCUSSION THREAD

**Discussion by acheong08** [acheong08/ChatGPT#1147] [Announcements]
None so far.

Future plans involve **opt in** data collection for **V1 only**. If anyone has suggestions on how to do this in a transparent way and privacy respecting way, leave a comment below.

Data wanted: Conversation data

Disclosure: Published publicly to GitHub. 

Usage: Training / Fine tuning AI models

Justification: LLaMA got leaked. We want to make an open source ChatGPT.

**Comment by acheong08** [comment]
For all the privacy advocates, I repeat, this will **not** be enabled by default

**Reply by acheong08** [reply]
On another note, most other public cloudflare bypass servers are known to collect your data by default with no option to disable.

**Comment by xloem** [comment]
This can store data permanently on arweave for free: https://github.com/xloem/pyarweave/blob/main/test_upload_bundlr.py . The http backend code can be a little flaky though; I’m not actively maintaining the library.

**Comment by acheong08** [comment]
ping

The implementation is done. Let me know any possible improvements to the system. It's currently done via a separate server and a postgres database

**Comment by bokesyo** [comment]
+1

**Comment by xloem** [comment]
How do we access the data?
Can you archive exchanges with ChatGPT Pro?

**Reply by acheong08** [reply]
I'm working on cleaning it up. Will post to huggingface once that's done
